packages/cf-style-provider/src/felaSnapshot.js

```diff
@@ -1,20 +1,28 @@
 import renderer from 'react-test-renderer';
-import { html as beautify } from 'js-beautify';
-import CleanCSS from 'clean-css';
 import felaTestContext from './felaTestContext';
 import createRenderer from './createRenderer';
 
-const cssCleaner = new CleanCSS({ format: 'beautify', level: 0 });
-
 export default component => {
   const felaRenderer = createRenderer({
     dev: true
   });
 
+  const prettifyFelaString = str =>
+    str
+      // adds extra lines before @media
+      .replace(/\@media/g, '\n\n@media')
+      // adds extra lines between keyframes and following selectors
+      .replace(/\}\.([a-zA-Z])/g, '}\n\n.$1')
+      // adds extra indentation and lines to nested selectors
+      .replace(
+        /\)\{(\.[\s\S]+?\})(\})/g,
```

Comment:
This will only work as long as Fela's formatting remains the same. Fragile as it is, it does generate some pretty good media query formats. Looks good to me.